sdk/python/setup.py

```diff
@@ -30,12 +30,12 @@
     'google-api-core>=1.31.5,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0',
     # `Blob.from_string` was introduced in google-cloud-storage 1.20.0
     # https://github.com/googleapis/python-storage/blob/master/CHANGELOG.md#1200
-    'google-cloud-storage>=1.20.0,<2',
+    'google-cloud-storage>=1.20.0,<3',
```

Comment:
Note that the only breaking change in `google-cloud-storage` major version 2.0.0 release version is dropping support for Python 2 (https://github.com/googleapis/python-storage/releases/tag/v2.0.0).

Reply:
`google-cloud-storage<2` had a lower upper bound on `google-auth` than `<3`, so this change is necessary for the `google-auth` relaxation to have an effect.